Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "bevy_fsl_box_frame"
-version = "0.3.0"
+version = "0.5.0"
```

Only need to bump once

---

```suggestion
version = "0.4.0"
```
Oh, right, good catch, I forgot I had already bumped it and bumped it again by mistake